### PR TITLE
wambo: init at 0.3.1

### PR DESCRIPTION
**Pull request by phip1611**
###### Description of changes

Add [wambo](https://crates.io/crates/wambo) at 0.3.1 to nixpkgs.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/wambo/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  cargoSha256 = "sha256-ghUdhWW5gURWxj/OhbcKPNeLzeJvndqAxEZmwKBATUk=";
+
+  meta = with lib; {
+    description = "All-in-one tool to convert decimal/bin/oct/hex and interpret bits as integers";
+    longDescription = ''
+      wambo is a binary that can easily shows you a numeric value in all important numeral systems
+      (bin, hex, dec) + interprets the input as both signed and unsigned values (from i8 to i64,
+      including f32 and f64). It also easily calculates you mibibytes to bytes, kilobytes to gibibytes,
+       and so on..
```
Comment (AndersonTorres) [inline pkgs/development/tools/wambo/default.nix:23]:
```suggestion
       and so on.
```

File: pkgs/development/tools/wambo/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    '';
+    homepage = "https://github.com/phip1611/wambo";
+    changelog = "https://github.com/phip1611/wambo/blob/v${version}/CHANGELOG.md";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ phip1611 ];
+  };
+}
+
```
Comment (AndersonTorres) [inline pkgs/development/tools/wambo/default.nix:31]:
```suggestion
```

File: pkgs/development/tools/wambo/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wambo";
+  version = "0.3.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-WZQgQmoFmsWLgPYRWonJmyKq9IIJ+a6J+O19XPppJG4=";
```
Comment (AndersonTorres) [inline pkgs/development/tools/wambo/default.nix:12]:
```suggestion
    hash = "sha256-WZQgQmoFmsWLgPYRWonJmyKq9IIJ+a6J+O19XPppJG4=";
```

File: pkgs/development/tools/wambo/default.nix [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  };
+
+  cargoSha256 = "sha256-ghUdhWW5gURWxj/OhbcKPNeLzeJvndqAxEZmwKBATUk=";
```
Comment (AndersonTorres) [inline pkgs/development/tools/wambo/default.nix:15]:
```suggestion
  cargoHash = "sha256-ghUdhWW5gURWxj/OhbcKPNeLzeJvndqAxEZmwKBATUk=";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by phip1611** [comment]
@AndersonTorres please re-check

**Review by AndersonTorres — APPROVED** [review]

**Pull request merged**